squad-server/log-parser/round-ended.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Matches when Map state Changes to PostMatch (ScoreBoard)
+ *
+ * Emits winner and loser from eventstore
+ *
+ * winner and loser may be null if the match ends with a draw
+ */
+export default {
+  regex:
+    /^\[([0-9.:-]+)]\[([ 0-9]*)]LogGameState: Match State Changed from InProgress to WaitingPostMatch/,
+  onMatch: (args, logParser) => {
+    const data = {
+      winner: logParser.eventStore.ROUND_WINNER ? logParser.eventStore.ROUND_WINNER : null,
```

Comment:
You could do `logParser.eventStore.ROUND_WINNER || null` here I think.

Reply:
I think they both would have the same effect. I prefer ect0s' code imo. More generally readable I find.